default.nix

```diff
@@ -1,3 +1,10 @@
-{
-  inherit (import ./lib) config create mount;
-}
+(import
+  (
+    let lock = builtins.fromJSON (builtins.readFile ./flake.lock); in
+    fetchTarball {
+      url = "https://github.com/edolstra/flake-compat/archive/${lock.nodes.flake-compat.locked.rev}.tar.gz";
+      sha256 = lock.nodes.flake-compat.locked.narHash;
+    }
+  )
+  { src = ./.; }
+).defaultNix.lib.body
```

Comment:
I think the intention was to only export `config` `create` and `mount` as official interfaces. Something like that would better fit the spirit and better compose with non-flake users:
```nix
{ lib ? import <nixpkgs/lib> }:
let my_lib = import ./lib { inherit lib; }; in
{
  inherit (my_lib) config create mount;
}
```